cli-output/src/cli_output.rs

```diff
@@ -1137,32 +1137,55 @@ fn show_votes_and_credits(
     Ok(())
 }
 
+macro_rules! choose_format {
```

Comment:
minor, but have a couple thoughts here:

1. bool args are typically (imo) not great. It seems reasonable we might want to support additional formats in the future. wdyt of using a passed enum for the format?
2. the name `choose_format` *to me* doesn't imply it does the formatting, but just chooses what format we will use. Maybe something like `format_as`? works better w/ above suggestion!

Reply:
ok. changed to enum and renamed to format_as, thanks for the suggestions